### PR TITLE
[Docs] Translate three docs

### PR DESCRIPTION
**Pull request by JetstramSam**
翻译了models.md 和 structures.md 和 transforms.md

### CODE REVIEW

File: docs/en/advanced_guides/models.md [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
 
-- algorithms, containing the full modules of a model and all sub-modules will be
-  constructed in algorithms.
-- backbones, containing the backbones for each algorithm, e.g. ViT for MAE, and Swim Transformer
-  for SimMIM.
-- necks, some specifial modules, such as decoder, appended directly to the output of the backbone.
-- heads, some specifial modules, such as mlp layers, appended to the output of the backbone or neck.
-- memories, some memory banks or queues in some algorithms, e.g. MoCov1/v2.
-- losses, used to compute the loss between the predicted output and the target.
+-算法，包括模型的全部模块和构造算法时需要用到的子模块。
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/models.md]:
```suggestion
- 算法，包括模型的全部模块和构造算法时需要用到的子模块。
```

File: docs/en/advanced_guides/models.md [hunk 2]
```diff
@@ -80,14 +80,10 @@ class BaseModel(_BaseModel):
...
-your algorithm, you should overwrite some abstract functions in the base model to get the correct results, which we will discuss in the
-following section.
-
-## Overview these abstract functions in base model
+正如上面代码所示，构造脊柱部分时需要配置，但是对颈部和头部而言这可有可无。除了构造算法之外，你还需要重写基础模型中的一些抽象函数才能得到正确结果，我们将在下一部分讨论这件事。
 
-The `forward` function is the entrance to the results. However, it is different from the default `forward` function in most PyTorch code, which
-only has one mode. You will mess all your logic in the `forward` function, limiting the scability. Just as shown in the code below, `forward` function in MMSelfSup has three modes, i) tensor, ii) loss and iii) predict.
+## 纵览基础模型中的抽象函数
+'forward'函数是结果的入口。然而，它和大多数Pytorch代码中只有一种模式的'forward'函数不同。在'forward'函数中你会逻辑混乱，这会限制粗糙度。正如下面代码所示，MMSelfSup中的'forward'函数有三种模式：张量，损失和预测。
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/models.md]:
```suggestion
`forward` 函数是结果的入口。然而，它和大多数 Pytorch 代码中只有一种模式的 `forward` 函数不同。在 `forward` 函数中你会逻辑混乱，这会限制粗糙度。正如下面代码所示，MMSelfSup 中的 `forward` 函数有三种模式：张量，损失和预测。
```

File: docs/en/advanced_guides/structures.md [hunk 3]
```diff
@@ -1,32 +1,26 @@
...
+  - [SelfSupDataSample中的定制化的属性](#SelfSupDataSample中的定制化的属性)
+  - [用MMSelfSup把数据打包给SelfSupDataSample](#用MMSelfSup把数据打包给SelfSupDataSample)
 
-The same as those in other OpenMMLab repositories, MMSelfSup defines a data structure, called `SelfSupDataSample`, which is used to receive and pass data during the whole training/testing process.
-`SelfSupDataSample` inherits the `BaseDataElement` implemented in [MMEngine](https://github.com/open-mmlab/mmengine).
-We recommend users to refer to [BaseDataElement](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/tutorials/data_element.md)
-for more in-depth introduction about the basics of `BaseDataElement`. In this tutorials, we mainly discuss some customized
-features in [SelfSupDataSample](mmselfsup.structures.SelfSupDataSample).
+像OpenMMLab中其他仓库一样，MMSelfSup也定义了一个数据结构，名为`SelfSupDataSample`,这个数据结构用于接收和传递整个训练和测试过程中的数据。
+`SelfSupDataSample`继承[MMEngine](https://github.com/open-mmlab/mmengine)中使用的`BaseDataElement`。如果需要深入了解`BaseDataElement`，我们建议参考[BaseDataElement](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/tutorials/data_element.md)。在这些教程中，我们主要讨论[SelfSupDataSample](mmselfsup.structures.SelfSupDataSample)中一些定制化的属性。
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/structures.md]:
```suggestion
`SelfSupDataSample`继承[MMEngine](https://github.com/open-mmlab/mmengine)中使用的`BaseDataElement`。如果需要深入了解`BaseDataElement`，我们建议参考[BaseDataElement](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/advanced_tutorials/data_element.md)。在这些教程中，我们主要讨论[SelfSupDataSample](mmselfsup.structures.SelfSupDataSample)中一些定制化的属性。
```

File: docs/en/advanced_guides/structures.md [hunk 4]
```diff
@@ -1,32 +1,26 @@
...
 
-## Customized attributes in SelfSupDataSample
+## SelfSupDataSample中的定制化的属性
+在MMSelfSup中，`SelfSupDataSample`将模型需要的所有信息（除了图片）打包，比如mask image modeling(MIM)中请求的`mask`和前置任务中的`pseudo_label`。除了提供信息，它还能接受模型产生的信息，比如预测得分。为实现上述功能，`SelfSupDataSample`定义以下五个属性：
 
-In MMSelfSup, except for images, `SelfSupDataSample` wraps all information required by models, e.g. `mask` requested by
-mask image modeling(MIM) and `pseudo_label` in pretext tasks. In addition to providing information, it can also accept
-information generated by models, such as the prediction score. To fulfill these functionalities described above, `SelfSupDataSample` defines five
-customized attributes:
+-gt_label（标签数据），包含图片的真实标签。
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/structures.md]: please check Chinese name of LabelData, InstanceData, BaseDataElement in MMEngine
https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/advanced_tutorials/data_element.md

File: docs/en/advanced_guides/transforms.md [hunk 5]
```diff
@@ -1,25 +1,23 @@
...
-  - [Overview of transforms](#overview-of-transforms)
-  - [Introduction of `MultiView`](#introduction-of-multiview)
-  - [Introduction of `PackSelfSupInputs`](#introduction-of-packselfsupinputs)
+- [转换](<>)
+  - [转换概述](#转换概述)
+  - [MultiView概论](#MultiView概论)
+  - [PackSelfSupInputs概论](#PackSelfSupInputs概论)
 
-## Overview of transforms
+## 转换概述
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/transforms.md]: 用 数据变换 这个翻译吧，比 转换 更加容易理解

File: docs/en/advanced_guides/transforms.md [hunk 6]
```diff
@@ -1,25 +1,23 @@
...
-`transforms`. There are three main categories of `transforms` in MMSelfSup:
+在[add_transforms](./add_transforms.md)中我们介绍了如何建`Pipeline`。`Pipeline`包括一系列的转换。MMSelfSup中转换主要分为三类：
 
-1. Transforms about processing the data. The unique transforms in MMSelfSup are defined in [processing.py](https://github.com/open-mmlab/mmselfsup/blob/1.x/mmselfsup/datasets/transforms/processing.py), e.g. `RandomCrop`, `RandomResizedCrop` and `RandomGaussianBlur`.
-   We may also use some transforms from other repositories, e.g. `LoadImageFromFile` from MMCV.
+1. 处理数据用到的转换。[processing.py](https://github.com/open-mmlab/mmselfsup/blob/1.x/mmselfsup/datasets/transforms/processing.py)中定义了独特的转换，比如`RandomCrop`, `RandomResizedCrop` and `RandomGaussianBlur`。我们也可以用其它仓库的转换，比如MMCV中的`LoadImageFromFile`。
 
-2. The transform wrapper for multiple views of an image. It is defined in [wrappers.py](https://github.com/open-mmlab/mmselfsup/blob/1.x/mmselfsup/datasets/transforms/wrappers.py).
+2.不同视角看同一照片的转换打包器。这个在[wrappers.py](https://github.com/open-mmlab/mmselfsup/blob/1.x/mmselfsup/datasets/transforms/wrappers.py)中定义过。
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/transforms.md]:
```suggestion
2. 不同视角看同一照片的转换打包器，这个定义在 [wrappers.py](https://github.com/open-mmlab/mmselfsup/blob/1.x/mmselfsup/datasets/transforms/wrappers.py)。
```

File: docs/en/advanced_guides/transforms.md [hunk 7]
```diff
@@ -35,10 +33,9 @@ In summary, we implement these `transforms` below. The last two transforms will
 |                             [`MultiView`](mmselfsup.datasets.transforms.MultiView)                              |               A wrapper for algorithms with multi-view image inputs                |
 |                           [`PackSelfSupInputs`](mmselfsup.datasets.PackSelfSupInputs)                           |         Pack data into a format compatible with the inputs of an algorithm         |
 
-## Introduction of `MultiView`
+## MultiView概论
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/transforms.md]:
```suggestion
## MultiView 简介
```

File: docs/en/advanced_guides/transforms.md [hunk 8]
```diff
@@ -68,13 +65,11 @@ pipeline = [
...
+这意味着有两个通信流水线，他们分别有两个角度和六个角度。在[imagenet_mocov1.py](https://github.com/open-mmlab/mmselfsup/blob/1.x/configs/selfsup/_base_/datasets/imagenet_mocov1.py)和 [imagenet_mocov2.py](https://github.com/open-mmlab/mmselfsup/blob/1.x/configs/selfsup/_base_/datasets/imagenet_mocov2.py) 和[imagenet_swav_mcrop-2-6.py](https://github.com/open-mmlab/mmselfsup/blob/1.x/configs/selfsup/_base_/datasets/imagenet_swav_mcrop-2-6.py)中有更多例子。
 
-## Introduction of `PackSelfSupInputs`
+## PackSelfSupInputs概论
 
-We build a class named [`PackSelfSupInputs`](mmselfsup.datasets.transforms.PackSelfSupInputs) to pack data into a format compatible with the inputs of an algorithm. This transform
-is usually put at the end of the pipeline like:
+我们建一个名为[`PackSelfSupInputs`](mmselfsup.datasets.transforms.PackSelfSupInputs)的类来将数据转换为能输入算法中的格式。这种转换通常在通信流水线的后面，就像这样：
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/transforms.md]:
```suggestion
我们建一个名为[`PackSelfSupInputs`](mmselfsup.datasets.transforms.PackSelfSupInputs)的类来将数据转换为能输入算法中的格式。这种转换通常在数据管道的最后，就像这样：
```
pipeline 的话，翻成 数据管道吧，全文可以检查一下

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

File: docs/en/advanced_guides/transforms.md (hunk 8 above)
Comment (JetstramSam) [reply docs/en/advanced_guides/transforms.md]: 改好了，那两个好像还是报错。commit次数有些多，需要的话我再开个分支重新PR。

Comment (fangyixiao18) [reply docs/en/advanced_guides/transforms.md]: > 改好了，那两个好像还是报错。commit次数有些多，需要的话我再开个分支重新PR。

没事，commit次数没关系

File: docs/en/advanced_guides/models.md [hunk 9]
```diff
@@ -41,11 +42,10 @@ described above.
...
 |     [`SwAV`](TODO)     |        [`ResNet`](TODO)         |      [`SwAVNeck`](TODO)      |          [`SwAVHead`](TODO)          |         [`SwAVLoss`](TODO)         |          N/A           |
 
-## Construct algorithms from sub-modules
+## 用子模块来构造算法
 
-Just as shown in above table, each algorithm is a combination of backbone, neck, head, loss and memories. You are free to use these existing modules to build your own algorithms. If some customized modules are required, you should follow [add_modules](./add_modules.md) to meet your own need.
-MMSelfSup provides a base model, called `BaseModel`, and all algorithms
-should inherit this base model. And all sub-modules, except for memories, will be built in the base model, during the initialization of each algorithm. Memories will be built in the `__init__` of each specific algorithm. And loss will be built when building the head.
+正如上表所述，每个算法都是主干，颈部，头部，损失和记忆的结合体。您可以从这些模块中任意选出若干部分来构建你自己的算法。如果需要定制化的模块，您可参考 [add_modules](./add_modules.md) 中的内容。
+MMSelfSup 提供一个基础模型,名为 `BaseModel` , 所以的算法都应该继承这个基础模型。而且所有子模块（除了记忆部分）在算法初始化时都基于这个基础模型。记忆部分在每个算法的 `__init__` 中被构造。损失部分在构造头部部分时被构造。
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/models.md]:
```suggestion
MMSelfSup 提供一个基础模型，名为 `BaseModel`，所以的算法都应该继承这个基础模型，而且所有子模块（除了记忆部分）在基础模型中进行初始化。记忆部分在对应算法的 `__init__` 中被构造。损失部分在头部部分初始化时被构造。
```

File: docs/en/advanced_guides/models.md [hunk 10]
```diff
@@ -105,13 +101,10 @@ def forward(self,
         raise RuntimeError(f'Invalid mode "{mode}".')
 ```
 
-- tensor, if the mode is `tensor`, the forward function will return the extracted features for images.
-  You should overwrite the `extract_feat` to implement your customized extracting process.
+- 张量，如果模式为 `tensor`，`forward` 函数就返回从图片提取到的特征。您应该重写其中的  `extract_feat`部分才能让定制化的提取过程有效。
 
-- loss, if the mode is `loss`, the forward function will return the loss between the prediction and the target.
-  You should overview the `loss` to implement your customized loss function.
+- 损失，如果模式为 `loss`，`forward` 函数就返回预测值与目标之间的损失。同样的，您应该重写其中的 `extract_feat` 部分才能让定制化的提取过程有效。
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/models.md]:
```suggestion
- 损失，如果模式为 `loss`，`forward` 函数就返回预测值与目标之间的损失。同样的，您应该重写其中的 `loss` 部分才能让定制化的提取过程有效。
```

File: docs/en/advanced_guides/models.md [hunk 11]
```diff
@@ -80,14 +80,10 @@ class BaseModel(_BaseModel):
...
-your algorithm, you should overwrite some abstract functions in the base model to get the correct results, which we will discuss in the
-following section.
-
-## Overview these abstract functions in base model
+正如上面代码所示，构造主干部分时需要配置，但是对颈部和头部而言这可有可无。除了构造算法之外，您还需要重写基础模型中的一些抽象函数才能得到正确结果，我们将在下一部分讨论这件事。
 
-The `forward` function is the entrance to the results. However, it is different from the default `forward` function in most PyTorch code, which
-only has one mode. You will mess all your logic in the `forward` function, limiting the scability. Just as shown in the code below, `forward` function in MMSelfSup has three modes, i) tensor, ii) loss and iii) predict.
+## 纵览基础模型中的抽象函数
+`forward` 函数是结果的入口。然而，它和大多数 Pytorch 代码中只有一种模式的 `forward` 函数不同。在 `forward` 函数中您会逻辑混乱，这会限制粗糙度。正如下面代码所示，MMSelfSup 中的 `forward` 函数有三种模式：张量，损失和预测。
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/models.md]:
```suggestion
`forward` 函数是结果的入口。然而，它和大多数 Pytorch 代码中只有一种模式的 `forward` 函数不同。MMSelfSup 把所有的逻辑都混杂在 `forward` 中，从而限制了该方法的可拓展性。正如下面代码所示，MMSelfSup 中的 `forward` 函数根据不同模式进行前向处理，目前共有三种模式：张量，损失和预测。
```

File: docs/en/advanced_guides/structures.md [hunk 12]
```diff
@@ -1,91 +1,82 @@
-# Structures
+# 结构
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/structures.md]: 数据结构

File: docs/en/advanced_guides/transforms.md [hunk 13]
```diff
@@ -35,10 +33,9 @@ In summary, we implement these `transforms` below. The last two transforms will
 |                             [`MultiView`](mmselfsup.datasets.transforms.MultiView)                              |               A wrapper for algorithms with multi-view image inputs                |
 |                           [`PackSelfSupInputs`](mmselfsup.datasets.PackSelfSupInputs)                           |         Pack data into a format compatible with the inputs of an algorithm         |
 
-## Introduction of `MultiView`
+## MultiView 简介
 
-We build a wrapper named [`MultiView`](mmselfsup.datasets.transforms.MultiView) for some algorithms e.g. MOCO, SimCLR and SwAV with multi-view image inputs. In the config file, we can
-define it as:
+我们为一些算法建名为 [`MultiView`](mmselfsup.datasets.transforms.MultiView) 的多角度照片输入的包装器，比如 MOCO, SimCLR和 SwAV。在配置文件中，我们能这样定义：
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/transforms.md]:
```suggestion
我们为一些算法定义了名为 [`MultiView`](mmselfsup.datasets.transforms.MultiView) 的多角度照片输入的包装器，比如 MoCo 系列，SimCLR，SwAV 等。在配置文件中，我们能这样定义：
```

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

**Review by fangyixiao18 — APPROVED** [review]
